### ISSUE TITLE
Weird heading spacing in bullet list

### ISSUE THREAD

**Issue by WilliamDev588** [chetachiezikeuzor/Yin-and-Yang-Theme#6]
Edit mode:
<img width="420" alt="Screen Shot 2021-06-06 at 22 14 20" src="https://user-images.githubusercontent.com/83950576/120929766-8df74d00-c714-11eb-95b9-ae805eb19fea.png">

Preview mode (bug):
<img width="532" alt="Screen Shot 2021-06-06 at 22 11 59" src="https://user-images.githubusercontent.com/83950576/120929697-3a84ff00-c714-11eb-865f-1923e07a24a3.png">

**Comment by chetachiezikeuzor** [comment]
Hello! This spacing does look pretty funky. However, I've tried it in my own note and it seems to look just fine. It's possible I'm just missing something on my end. Is there anything else in this note of yours? And also, when did you install the theme? I've made some updates here and there and although I don't remember specifically dealing with bullet points, it's possible I may have. 

Fonts Used: 
- Galaxie Copernicus 
- Karla

<img width="552" alt="Screen Shot 2021-06-06 at 10 58 53 AM" src="https://user-images.githubusercontent.com/79069364/120931311-a2a4f800-c6b6-11eb-86e6-2508c6642d97.png">
<img width="552" alt="Screen Shot 2021-06-06 at 10 58 24 AM" src="https://user-images.githubusercontent.com/79069364/120931313-a33d8e80-c6b6-11eb-9c5a-c6a82608b1ab.png">
